src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -601,6 +611,11 @@
             "description": "Called when Step Functions Workflow Studio is closed",
             "metadata": [{ "type": "didSave" }]
         },
+        {
+            "name": "appcomposer_resourceCount",
+            "description": "Called to count resources in a template when it is opened",
+            "metadata": [{ "type": "resourceCounts" }, { "type": "totalResourceCount" }]
```

Comment:
Can this be on the `appcomposer_openTemplate` event? That is the recommended practice. There is no need to add a new event for every bit of info that is needed. Events should map to "actions". Events can have arbitrary metadata. Adding a new event just to track metadata should be avoided.

Reply:
At this point in the code I don't have access to the `hasChatAuth` param required for `appcomposer_openTemplate`. We are getting this event from `FigAssets`, which is environment-agnostic and doesn't have access to VSCode actions

Reply:
This PR is mainly to pass that data along. I could rename the `TEMPLATE_OPENED` event to `COUNT_RESOURCES` - the only reason I'm using `TEMPLATE_OPENED` is that's how it's set up in `FigAssets`